docs/led-array-controller.md

```diff
@@ -31,20 +31,20 @@ node eg/led-array-controller.js
 
 
 ```javascript
-var five = require("johnny-five");
-var board = new five.Board();
+const {Board, Leds, Sensor} = require("johnny-five");
+const board = new Board();
 
 board.on("ready", function() {
-  var leds = new five.Leds([2, 3, 4, 5, 6]);
-  var pot = new five.Sensor("A0");
+  const leds = new Leds([2, 3, 4, 5, 6]);
+  const pot = new Sensor("A0");
 
-  pot.scale([-1, 4]).on("change", function() {
-    var lastIndex = Math.round(this.value);
+  pot.scale([-1, 4]).on("change", () => {
```

Comment:
So I was thinking that originally but in this case I noticed we only pass the `rawValue` of the Sensor so it confused me as I wasn't able to get the scaled value as far as I could tell without using the object itself.

Reply:
I think this is one of the things driving @ajfisher's audit of parameters passed to callbacks. For now, I think we just roll with it and update the docs as best we can. We will hammer out the callback parameters in another effort, especially since no matter what we do it will be a breaking change.